x-pack/platform/test/api_integration/services/ml/index.ts

```diff
@@ -5,14 +5,14 @@
  * 2.0.
  */
 
-import { FtrProviderContext } from '../../functional/ftr_provider_context';
+import { FtrProviderContext } from '../../ftr_provider_context';
 
-import { MachineLearningAPIProvider } from '../../functional/services/ml/api';
-import { MachineLearningCommonAPIProvider } from '../../functional/services/ml/common_api';
-import { MachineLearningCommonConfigsProvider } from '../../functional/services/ml/common_config';
-import { MachineLearningSecurityCommonProvider } from '../../functional/services/ml/security_common';
-import { MachineLearningTestExecutionProvider } from '../../functional/services/ml/test_execution';
-import { MachineLearningTestResourcesProvider } from '../../functional/services/ml/test_resources';
+import { MachineLearningAPIProvider } from './api';
+import { MachineLearningCommonAPIProvider } from './common_api';
+import { MachineLearningCommonConfigsProvider } from './common_config';
+import { MachineLearningSecurityCommonProvider } from './security_common';
+import { MachineLearningTestExecutionProvider } from './test_execution';
+import { MachineLearningTestResourcesProvider } from './test_resources';
```

Comment:
I'm not deleting original services under `x-pack/test/functional` to avoid updating imports in functional tests. We will delete them after all tests are relocated